docs/NativeCodeGeneration.md

```diff
@@ -23,9 +23,13 @@ image that can be loaded and used as an eBPF program.
 
 # Native Code Generation Pipeline
 
+The following diagram shows the steps in the pipeline:
+
+![eBPF Driver Toolchain](DriverToolchain.png)
```

Comment:
This link also looks broken.

Reply:
looks like this is not showing as this PR is not yet merged to main. Please ignore.